script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_2.txt

```diff
@@ -28,7 +28,7 @@ It's really weird to go into a girl's room and ask her to let you watch her TV.
 Even Kohaku-san is looking at me in disgust――――or maybe not?
 <Page9>
 "Now that you mention it, that's true.
-You've been living in the Arima house until now. The atmosphere in this mansion must have been quite heavy all of a sudden, right?"
+The heavy atmosphere in this mansion must have been quite a shock to the system after living in the Arima household"
```

Comment:
Missing period. Also do you think the consolidation works better? I'd just left that in as context, but if this works better than
> You've been residing at the Arima household up until now after all. The heavy atmosphere in this mansion must have come as quite a shock.//'Residing' a bit formal to account for keigo
> 志貴さんは今まで有間のお家で暮らしていたんですもの。いきなりこの屋敷の空気は重かったでしょう？

then go for it

Reply:
I like ross' new version the most